File: Javascript/URI/01-Iniciantes/1000-1020/1011.js
sphereSize = SphereSize();
sphereVolume = CalculateSphereSize(sphereSize);
console.log("VOLUME = " + sphereVolume.toFixed(3));
alert("VOLUME = " + sphereVolume.toFixed(3));

function SphereSize()
{
    sphereSize = parseFloat(prompt("Digite o tamanho da espera!"));
    return sphereSize;
}
function CalculateSphereSize(sphereSize)
{
    const pi = 3.14159, sphereFormula = 4/3.0;
    let sphereVolume;
    sphereVolume = sphereFormula * pi * Math.pow(sphereSize, 3);
    return sphereVolume;
}

File: Cplusplus/URI/01-iniciantes/02-Nivel/1011.cpp
#include <iostream>
#include <iomanip>
#include <math.h>

using namespace std;
double SetSphereSize();
double CalculateSphereVolume(double sphereSize);

int main()
{
    double sphereSize; 
    sphereSize = SetSphereSize();    
    cout << "VOLUME = " << fixed << setprecision(3) << CalculateSphereVolume(sphereSize) << endl;    
    return 0;
}
double SetSphereSize()
{
    double sphereSize;
    cout << " Digite o raio da esfera." << endl;
    cin >> sphereSize >> setprecision(2) >> fixed;
    return sphereSize;
    
}
double CalculateSphereVolume(double sphereSize)
{
    double pi = 3.14159, formulaConstant = 4 / 3.0, sphereVolume;
    sphereVolume = formulaConstant * pi * pow(sphereSize, 3);
    return sphereVolume;
}